pkg/apis/scheduling/v1alpha1/replicaschedulingpreference_types.go

```diff
@@ -45,6 +45,14 @@ type ReplicaSchedulingPreferenceSpec struct {
 	// +optional
 	Rebalance bool `json:"rebalance,omitempty"`
 
+	// If set to true then the target kind (FederatedDeployment or FederatedReplicaset) placement
+	// will be intersected with Clusters of the RSP to determine the result clusters of the
+	// federated resource.
+	// if set false or not defined, Clusters in the RSP overriding the placment in the federated
+	// resource.
+	// +optional
+	PlacementIntersection bool `json:"placementIntersection,omitempty"`
```

Comment:
ClusterSelector is a little misleading I guess. For a federated resource, we have 
```
spec:
  placement:
    clusters:
        ...
    clusterSelector:
        ...
```
I use Placement here because it includes both `clusters` and `clusterSelector`.

Any comments?